README.md

```diff
@@ -57,6 +57,10 @@ public static void main(String[] args) {
 
 ```
 
+#### META-INF
+
+To ensure Jersey/HK2 SPIs will use the proper ServiceLocationGenerator, simply copy the META-INF folder into your root project's resources folder.
```

Comment:
Doing this is fair enough but how is it different from adding `jersey2-guice-spi` to the classpath. Both are subject to classpath ordering.